backend/core/src/migration/1675104670318-remove-household-member-phone-and-email.ts

```diff
@@ -0,0 +1,26 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class removeHouseholdMemberPhoneAndEmail1675104670318 implements MigrationInterface {
+    name = 'removeHouseholdMemberPhoneAndEmail1675104670318'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "household_member" DROP COLUMN "no_email"`);
+        await queryRunner.query(`ALTER TABLE "household_member" DROP COLUMN "no_phone"`);
+        await queryRunner.query(`ALTER TABLE "household_member" DROP COLUMN "email_address"`);
+        await queryRunner.query(`ALTER TABLE "household_member" DROP COLUMN "phone_number"`);
+        await queryRunner.query(`ALTER TABLE "household_member" DROP COLUMN "phone_number_type"`);
+        await queryRunner.query(`ALTER TABLE "listings" ALTER COLUMN "afs_last_run_at" SET DEFAULT '1970-01-01'`);
```

Comment:
See PR description -- these are some unexpected changes to the listings table

Reply:
I am also unsure why this is happening. When I run it locally on the main branch this also happens to me. I think it might be because of the type we are using "timestamp with time zone". @YazeedLoonat do you know of a way we could prevent this? I noticed these are the only two fields with `TIMESTAMP WITH TIME ZONE DEFAULT` and I'm guessing when we run the migrations locally it's different timezone based on who ran it.

Reply:
@ludtkemorgan I'm also getting local changes when I run `generate:client` which seems unusual, thinking that just maybe needs to be committed?